plugins/api-docs/README.md

```diff
@@ -23,6 +23,70 @@ Other formats are displayed as plain text, but this can easily be extended.
 To fill the catalog with APIs, [provide entities of kind API](https://backstage.io/docs/features/software-catalog/descriptor-format#kind-api).
 To link that a component provides or consumes an API, see the [`providesApis`](https://backstage.io/docs/features/software-catalog/descriptor-format#specprovidesapis-optional) and [`consumesApis`](https://backstage.io/docs/features/software-catalog/descriptor-format#specconsumesapis-optional) properties on the Component kind.
 
+## Implementing OAuth 2 Authorization Code flow with Swagger UI
+
+### OAuth2 `redirect_uri` route
+
+The Swagger UI package by default expects to have a route to `/oauth2-redirect.html` which processes
+the redirect callback for the OAuth2 Authorization Code flow.
+
+A copy of [oauth2-redirect.html](https://github.com/swagger-api/swagger-ui/blob/master/dist/oauth2-redirect.html)
+has been added to the default Backstage `app/public/` directory in order to enable Swagger UI to complete
+this redirection.
+
+### Configuring your OAuth2 Client
+
+You'll need to make sure your OAuth2 client has been registered in your OAuth2 Authentication Server (AS)
+with the appropriate `redirect_uris`, `scopes` and `grant_types`. For example, if your AS supports
+the [OAuth 2.0 Dynamic Client Registration Protocol](https://tools.ietf.org/html/rfc7591), an example
+POST request body would look like this:
+
+```json
+{
+    "client_name": "Example Backstage api-docs plugin Swagger UI Client",
+    "redirect_uris": [
+        "https://www.getpostman.com/oauth2/callback",
+        "http://localhost:3000/oauth2-redirect.html"
+        "https://<yourhost>/oauth2-redirect.html"
+    ],
+    "scope": "read_pets write_pets",
+    "grant_types": [
+        "authorization_code"
+    ]
+}
+```
+
+The above `redirect_uris` are:
+
+- [Postman](https://www.postman.com/) testing: `https://www.getpostman.com/oauth2/callback`
+- Local Backstage app development: `http://localhost:3000/oauth2-redirect.html`
+- Backstage app production: `https:<yourhost>/oauth2-redirect.html`
```

Comment:
```suggestion
- Backstage app production: `https://<yourhost:port>/oauth2-redirect.html`
```

for consistency with example above?

Reply:
Oh yeah for the '//' but I think not for the port as I would expect a production host to be running on 443.

Reply:
resolved in next commit